corehq/form_processor/tests/test_usercase_dbaccessor.py

```diff
@@ -28,8 +30,15 @@ def setUp(self):
         factory = CaseFactory(domain='foo')
         factory.create_case(case_type=USERCASE_TYPE, owner_id=self.user._id, case_name='bar',
                             update={'hq_user_id': self.user._id})
+        print(factory.create_case(case_type=USERCASE_TYPE, owner_id=self.web_user._id, case_name='web',
+                            update={'hq_user_id': self.web_user._id}))
 
     def test_get_usercase(self):
         usercase = self.accessor.get_case_by_domain_hq_user_id(self.user._id, USERCASE_TYPE)
         self.assertIsNotNone(usercase)
         self.assertEqual(usercase.name, 'bar')
+
+    def test_get_web_user_usercase(self):
```

Comment:
I don't think this test is necessary since the db accessor isn't aware of the user type at all and only uses the user ID and case type for the lookup.